service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -125,6 +127,9 @@ if [[ -n "${TERRA_WORKSPACE}" ]]; then
   sudo -u "${JUPYTER_USER}" sh -c "terra workspace set --id=${TERRA_WORKSPACE}"
 fi
 
+# Generate cromwell.config
+sudo -u "${JUPYTER_USER}" sh -c "terra cromwell generate-config --dir=/home/${JUPYTER_USER}/cromwell"
```

Comment:
I put this line here because this generation process required a workspace set. The workspace set at L127.

Reply:
put the cromwell.conf in the home dir, so remove the -dir specification and also remove hte line where cromwell dir is created.